WordPress/src/main/java/org/wordpress/android/ui/accounts/LoginActivity.java

```diff
@@ -709,6 +709,25 @@ public void gotXmlRpcEndpoint(String inputSiteAddress, String endpointAddress) {
         }
     }
 
+    @Override public void gotRestEndpoint(String inputSiteAddress, String endpointAddress) {
+        CustomTabsIntent intent = new CustomTabsIntent.Builder()
```

Comment:
Nitpick: Since we already use a `CustomTabsIntent` in `showWPcomLoginScreen`, would it make sense to add a function to show a url? ie: `showCustomTabsIntent(uri: Uri)`